libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -724,11 +723,32 @@ void CodeTransform::visitExpression(Expression const& _expression)
 
 void CodeTransform::visitStatements(vector<Statement> const& _statements)
 {
+	// Workaround boost bug:
+	// https://www.boost.org/doc/libs/1_63_0/libs/optional/doc/html/boost_optional/tutorial/gotchas/false_positive_with__wmaybe_uninitialized.html
+	boost::optional<AbstractAssembly::LabelID> jumpTarget = boost::make_optional(false, AbstractAssembly::LabelID());
+
 	for (auto const& statement: _statements)
 	{
 		freeUnusedVariables();
+
+		const FunctionDefinition* functionDefinition = boost::get<FunctionDefinition>(&statement);
```

Comment:
```suggestion
		FunctionDefinition const* functionDefinition = boost::get<FunctionDefinition>(&statement);
```

Reply:
Should I just use `auto functionDefinition ...`?

Reply:
Fine with me!